minidns-core/src/main/java/de/measite/minidns/DNSClient.java

```diff
@@ -154,4 +125,20 @@ public static synchronized void addDnsServerLookupMechanism(DNSServerLookupMecha
     public static synchronized boolean removeDNSServerLookupMechanism(DNSServerLookupMechanism dnsServerLookup) {
         return LOOKUP_MECHANISMS.remove(dnsServerLookup);
     }
+
+    public boolean isAskForDnssec() {
+        return askForDnssec;
+    }
+
+    public void setAskForDnssec(boolean askForDnssec) {
+        this.askForDnssec = askForDnssec;
+    }
+
+    public boolean isDisableResultFilter() {
+        return disableResultFilter;
+    }
+
+    public void setDisableResultFilter(boolean disableResultFilter) {
+        this.disableResultFilter = disableResultFilter;
+    }
```

Comment:
Ok, it would enable querying dnssec records, why not queryDnssec(Enabled) or retrieveDnssec? That way it would be clear that it enables the query part and not the verification part. Would that make sense? (Although this is far from important, I just think isAsk does not sound right, and I'm doing a second diff-based read-through)